Neos.ContentRepository.Core/Classes/Projection/Workspace/Workspace.php

```diff
@@ -24,31 +24,72 @@
  *
  * @api
  */
-class Workspace
+final readonly class Workspace
 {
     /**
      * This prefix determines if a given workspace (name) is a user workspace.
      */
     public const PERSONAL_WORKSPACE_PREFIX = 'user-';
```

Comment:
this can also be removed or deprecated?

Reply:
good call, I'll move it to the only usage (`Workspace::isPersonalWorkspace()`)